hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/HoodieMetaSyncOperations.java

```diff
@@ -89,6 +89,15 @@ default void dropPartitions(String tableName, List<String> partitionsToDrop) {
 
   }
 
+  /**
+   * Get location for the table in the metastore.
+   * @param tableName
+   * @return
+   */
+  default Option<String> getMetastoreLocation(String tableName) {
+    return Option.empty();
+  }
+
```

Comment:
Same here.  This is only used in the tests.  Could you avoid adding the new API?

Reply:
Ok.